chatgpt-shell.el

```diff
@@ -199,28 +199,22 @@ Can be used compile or run source block at point."
                                   (cons (const primary-action) (function :tag "Action:"))))
   :group 'chatgpt-shell)
 
-(defcustom chatgpt-shell-model-versions
+(defcustom chatgpt-shell-models
```

Comment:
The reason I change this is [this comment](https://github.com/xenodium/chatgpt-shell/issues/217#issuecomment-2487452256)